File: Hash Table/705. Define HashSet/hash_set.cpp
class MyHashSet
{
private:
    vector<bool> hash_set;

public:
    MyHashSet()
    {
        hash_set.resize(1000001, false);
    }

    void add(int key)
    {
        hash_set[key] = true;
    }

    void remove(int key)
    {
        hash_set[key] = false;
    }

    bool contains(int key)
    {
        return hash_set[key];
    }
};

File: 4. Hash Table/705. Define HashSet/hash_set.py
class MyHashSet:
    def __init__(self):
        self.hash_set = [False] * 1000001

    def add(self, key: int) -> None:
        self.hash_set[key] = True

    def remove(self, key: int) -> None:
        self.hash_set[key] = False

    def contains(self, key: int) -> bool:
        return self.hash_set[key]
